### ISSUE TITLE
[Accessibility]A11y_VSCode_WelcomeScreen_ScreenReader: Unnecessary list markup present for the static content present under "Customize" heading.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52436]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : HighSierra10.13.5

**Additional Details:** 
MAS Violated : MAS 1.3.1
ScreenReader : VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Welcome screen.
3. Read all the content present under the heading "Customize".

**Actual:**
VoiceOver reads content of -"Tools and Languages ...", "Settings and Keybindings.." as list. These are headings and only links present in between the content can be considered in a list. Unnecessary markup of lists is present for static content.

**Expected:**
VoiceOver should read the headings separately and list markup should be removed for the static content present.
Lists can be formed out of links given inside above mentioned headings.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will come across unnecessary markup of lists for the static content present on the page which is not actionable.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_WelcomeScreen_VoiceOver_UnnecessaryListMarkup.pptx](https://github.com/Microsoft/vscode/files/2118835/A11y_VSCode_WelcomeScreen_VoiceOver_UnnecessaryListMarkup.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51986
